### ISSUE TITLE
Add Contact validation don't recognize corrections

### ISSUE THREAD

**Issue by dpp-gerrit** [saros-project/saros#79]
* Open HTML View
* Connect with an account
* Press "Add Contact.."
* While typing, add an syntax error in the field XMPP/JABBER ID f.e. whitespaces: "test with space@<span></span>ex.de"

The error is correctly recognized: A message is promted and the add button is disabled. However, if you correct the input by just removing the whitespaces, the message remains and the add button lookes still disabled (though it actualy isn't anymore). Only after hitting any letter it gets updated (enter doesnt work).

Reported by: MatthiasBohnstedt

Original Ticket: [dpp/bugs/886](https://sourceforge.net/p/dpp/bugs/886)

**Comment by fzieris** [comment]
The bug described above appears to pertain to a previous version of the HTML GUI. The current HTML GUI shows no such error message (and is erroneous in other ways -- but this deserves new issue(s)).
